src/Raven.Server/Documents/Handlers/HiLoHandler.cs

```diff
@@ -53,7 +53,7 @@ public async Task GetNextHiLo()
                 var tag = GetQueryStringValueAndAssertIfSingleAndNotEmpty("tag");
                 var lastSize = GetLongQueryString("lastBatchSize", false) ?? 0;
                 var lastRangeAt = GetStringQueryString("lastRangeAt", false);
-                var identityPartsSeparator = GetStringQueryString("identityPartsSeparator", false) ?? "/";
+                var identityPartsSeparator = GetCharQueryString("identityPartsSeparator", false) ?? Database.IdentityPartsSeparator;
```

Comment:
Need to think if this will not cause issues when 4.2 client will talk with 5.x server

Reply:
How could it? it will just default to the usual one, no?

Reply:
Maybe a problem if you are using a multi char string, you mean? Very unlikely